config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -64,10 +64,10 @@
 # Disable partial inserts.
 # This default means that all columns will be referenced in INSERT queries
 # regardless of whether they have a default or not.
-# Rails.application.config.active_record.partial_inserts = false
+Rails.application.config.active_record.partial_inserts = false
 #
 # Protect from open redirect attacks in `redirect_back_or_to` and `redirect_to`.
-# Rails.application.config.action_controller.raise_on_open_redirects = true
+Rails.application.config.action_controller.raise_on_open_redirects = true
```

Comment:
This means that `redirect_to` will be very strict and won't let you redirect to a different domain. I've updated the code that this will affect. It should be safe.